app/controller_services/shopping_list_items_controller/update_service.rb

```diff
@@ -25,7 +25,6 @@ def perform
       aggregate_list_item = nil
       ActiveRecord::Base.transaction do
         list_item.update!(params)
-        shopping_list.touch
```

Comment:
Should no longer be needed but we'll double check at the point where we make these tests pass again.